stdlib/public/core/RangeReplaceableCollection.swift

```diff
@@ -936,12 +950,15 @@ where Self: BidirectionalCollection, SubSequence == Self {
   public mutating func removeLast(_ k: Int) {
     if k == 0 { return }
     _precondition(k >= 0, "Number of elements to remove should be non-negative")
-    _precondition(count >= k,
-      "Can't remove more items from a collection than it contains")
     if _customRemoveLast(k) {
       return
     }
     let end = endIndex
+    guard let start = index(end, offsetBy: -k, limitedBy: startIndex)
+    else {
+      _preconditionFailure(
+        "Can't remove more items from a collection than it contains")
+    }
     removeSubrange(index(end, offsetBy: -k)..<end)
```

Comment:
I think this should be using `start`, too:
```suggestion
    removeSubrange(start..<end)
```

Reply:
Thanks for the catch!